site/content/integrate/apps/interactivity/_index.md

```diff
@@ -39,6 +39,7 @@ All fields include ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-
 | `name`        | string    | Key to use in the values field of the call.            |
 | `type`        | FieldType | The type of the field.                                 |
 | `is_required` | bool      | (Optional) Whether the field needs to be filled.       |
+| `multiselect` | bool      | (Optional) Whether a select field is multiselect.      |
```

Comment:
1/5 the sentence seems weird to me. Does it make sense to be more descriptive?

Reply:
@hanzei Maybe something like "True when multiple values are able to be selected."

Reply:
I've thought through multiple variances of this description. The two things I think it should portray are 1) this is for select fields and 2) allows multiple values to be selected.  

The `bool` type lets the caller know that it is a true/false type.

How about `(Optional) Whether a select field allows multiple values to be selected.`?